### PR TITLE
feat: add variables for email template

### PR DESCRIPTION
**Pull request by tmilost**
### Description:

User can configure in global variable, what is displayed in email template

### Checklist:

_Make sure you tick all the boxes below if they are true or do not apply before you ask for review_

- [ʕノ•ᴥ•ʔノ ︵ ┻━┻] I have performed a self-review of my code
- [ʕノ•ᴥ•ʔノ ︵ ┻━┻] I have made my code as simple as possible
- [ʕノ•ᴥ•ʔノ ︵ ┻━┻] have added relevant tests for my changes and the code coverage has not dropped substantially
- [ʕノ•ᴥ•ʔノ ︵ ┻━┻] I have removed all commented code
- [ʕノ•ᴥ•ʔノ ︵ ┻━┻] I have updated the documentation in all relevant places (Javadoc, Swagger, MDs...)
- [ʕノ•ᴥ•ʔノ ︵ ┻━┻] I have described the PR and added a meaningful title in the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format

### CODE REVIEW

**Comment by tmilost** [comment]
@jungwire Feel free to change whatever you think is necessary

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -69,6 +69,8 @@ spring:
 negotiator:
   molgenis-url: ""
   frontend-url: ""
+  emailYoursSincerelyText: ""
+  emailHelpdeskHref: ""
```
Comment (RadovanTomik) [inline src/main/resources/application.yaml:73]: please set it to BBMRI-ERIC by default and remove it from other property files

**Review by RadovanTomik — CHANGES_REQUESTED** [review]
were the changes made discussed with the person who requested these changes? There is no issue linked. Maybe add some tests or enable uploading a fully customized template file.

**Comment by jungwire** [comment]
merged the documentation branch in to this one

**Review by RadovanTomik — CHANGES_REQUESTED** [review]
please just set the email config variables in application.yml with defuault values

**Pull request merged**